src/jit/compiler.h

```diff
@@ -1429,16 +1429,15 @@ struct fgArgTabEntry
         return (regNumber)regNums[i];
     }
 
-    __declspec(property(get = getIsSplit, put = setIsSplit)) bool isSplit;
-    bool getIsSplit()
+    bool GetSplit()
```

Comment:
`IsSplit`?

Reply:
how about set function?

SetIsSplit?

Reply:
Keep it `SetSplit`.

Usually, we use `Is` prefix instead of `Get` for bool getters and keep `Set`, e.q.:
```
    bool IsStkAlignmentDone() const
    {
        return alignmentDone;
    }

    void SetStkAlignmentDone()
    {
        alignmentDone = true;
    }

```


Do you know that you can clone https://github.com/dotnet/jitutils and run the format fix locally? I am not sure that it is written anywhere.
```
pushd C:\git\tools\jitutils\bin
jit-format.bat --fix -c F:\git\coreclr
```

Reply:
thanks, I didn't know

Reply:
I tried jitutils on ubuntu linux 16.04. It generates too many differences. Almost every single file in jit directory is modified.

Reply:
That is very strange, it should be like this.
We run the same task in CI to check formatting (without `--fix`).
I have tried to run it on my Ubuntu 18 x64 machine and it was ok.

Could you show of the diffs? jit-format uses https://github.com/dotnet/coreclr/blob/master/src/jit/.clang-format for the rules, probably you have it somewhere in global space that overruns jit settings?

Reply:
this might explain it: 

sinankaya@sinanubuntu1604:~/workspace/coreclr/src/jit$ clang-format --version
clang-format version 10.0.0-svn373409-1~exp1+0~20191002001027.2519~1.gbpab8d9f (trunk)

Reply:
https://gist.github.com/franksinankaya/e9837f5a95a9d647970e0878c3bb5912

Reply:
@jashook could you please take a look? You know jit-format better than I.